observability/distr_tracing/distr-tracing-tempo-installing.adoc

```diff
@@ -1,11 +1,16 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="dist-tracing-tempo-installing"]
-= Installing
+= Installing the {DTShortName}
 include::_attributes/common-attributes.adoc[]
 :context: dist-tracing-tempo-installing
 
 toc::[]
 
+[TIP]
+====
+For installing the deprecated and soon to be removed {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing.adoc#dist-tracing-jaeger-installing[Installing] in the {JaegerShortName} documentation.
```

Comment:
> the deprecated and soon to be removed 

Same comment here.

Reply:
Yes, because Tempo is the replacement product for Jaeger, so readers can confuse the docs and may be looking at the wrong product.